package.json

```diff
@@ -144,16 +145,16 @@
   },
   "license": "MIT",
   "scripts": {
-    "postinstall": "./tasks/checkDependencies.js",
-    "lint": "./tasks/lint.js",
-    "e2e": "./tasks/lazyInstallVendor e2e && cypress run",
+    "postinstall": "node ./tasks/checkDependencies.js",
+    "lint": "eslint src/app src/helpers --ext .js --quiet --cache",
```

Comment:
eslint with cache option is pretty fast from the second rund. If you need fast even first run, you need to use some cross-platform file finding, not using bash.

Reply:
this version of eslint is now faster than our script :+1: